src/mol-plugin-ui/viewport/screenshot.tsx

```diff
@@ -120,11 +131,15 @@ function CameraInfo({ plugin }: { plugin: PluginContext }) {
     }, [plugin]);
 
     const state = plugin.canvas3d?.camera.state;
+    const fovAdjusted = fovAdjustedCameraPosition(state);
 
     return <div>
-        <CameraInfoSection title='Position'>
+        <CameraInfoSection title='Ref. Position'>
             {renderVector(state?.position)}
         </CameraInfoSection>
+        <CameraInfoSection title='FoV Adj. Pos.'>
+            {renderVector(fovAdjusted)}
+        </CameraInfoSection>
```

Comment:
Again, I believe the nomenclature here is misleading (see comment above).

Perhaps, the Molstar's internal camera position could be 'Position' and the position for MVS could be 'FOV-normalized position'. (and maybe mention MVS in help message/tooltip)

Reply:
A few lines below, you are showing camera direction vector. It would be nice to show it normalized. (unless it's normalized, it's not clear from which camera position it for calculated)